inference/api_openai_backend/query_client.py

```diff
@@ -75,8 +66,8 @@ async def stream(self, model: str, prompt: Prompt, request_id: str):
             prompt=prompt,
             request_id=request_id,
             async_iterator=deploy_handle.options(stream=True)
-            .stream_response.options(stream=True, use_new_handle_api=True)
-            .remote(prompt_content, gen_config),
+            .openai_call.options(stream=True, use_new_handle_api=True)
```

Comment:
need indent the two lines

Reply:
Indent cannot be added, lint will remove it.